src/libmongoc/src/mongoc/mongoc-cluster-private.h

```diff
@@ -215,6 +215,23 @@ _mongoc_cluster_get_auth_cmd_x509 (const mongoc_uri_t *uri,
                                    bson_t *cmd /* OUT */,
                                    bson_error_t *error /* OUT */);
 
+bool
```

Comment:
It would be nice to make these accessors instead of predicates-- although this is a little tider, the ability to /also/ fetch the server API in one step would be pleasant. Maybe something to do after this is "done"!